apps/dashboard/app/new/create-semantic-cache.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { toast } from "@/components/ui/toaster";
+import { trpc } from "@/lib/trpc/client";
+import { PostHogEvent, PostHogIdentify } from "@/providers/PostHogProvider";
+import { useUser } from "@clerk/nextjs";
+import { useRouter } from "next/navigation";
+import { z } from "zod";
+import { generateSemanticCacheDefaultName } from "../(app)/semantic-cache/new/util/generate-semantic-cache-default-name";
+
+type Props = any;
+
+export const CreateSemanticCacheButton: React.FC<Props> = () => {
+  const { user, isLoaded } = useUser();
+  const router = useRouter();
+
+  if (isLoaded && user) {
+    PostHogIdentify({ user });
```

Comment:
We don't need this, A user is already identified when they log in to the application. 

It's also expensive to load the entire user.